### DISCUSSION TITLE
Step-by-step guide. Step 1 is bit unnecessary

### DISCUSSION THREAD

**Discussion by gyroscopes** [dojo-engineering/feedback#7] [Docs]
Firstly congratulations on the payment flow graph on this page
https://docs.dojo.tech/docs/accept-payments/checkout-page/step-by-step-guide

It shows clearly what is going on. For me this actually was a major factor in choosing Dojo.
After a quick look around the main website, after seeing the pricing the next decision was to check what the API was like.
I could see C# was exampled and the graph showed the API should be straightforward.
Good graph.

But......when you go to step one. 

```
function myFunction() {
  // POST
  fetch('/checkout', {

    // Declare what type of data we're sending
    headers: {
      'Content-Type': 'application/json'
    },

    // Specify the method
    method: 'POST',
    mode: 'no-cors',

    // A JSON payload
    body: JSON.stringify({
      "greeting": ""
    })
  })
```

With the first few seconds I've spotted its just a URL call.
I'm then looking for a good minute more assuming its more complex there must be a reason why they did it that way.

Its just a URL call. It could be HTML link. Its not confusing really its just odd to introduce the API with that way. Step one is almost not required. 
Hope that makes sense.

**Comment by RomanKondratyev** [comment]
Well, I agree. @monica-m-ps please
